content/docs/connect/connect-from-any-app.md

```diff
@@ -21,22 +21,22 @@ You can obtain the connection details you require from the **Connection Details*
 
 ![Connection details widget](/docs/connect/connection_details.png)
 
-You can enable the **Pooler** option if you require a pooled connection string. The **Pooler** option adds `-pooler` to the hostname, which tells Neon to use a pooled connection. For more information, see [Connection pooling](/docs/connect/connection-pooling#connection-pooling).
+Neon supports pooled and direct connections to the database. Use a pooled connection string if your application uses a high number of concurrent connections. For more information, see [Connection pooling](/docs/connect/connection-pooling#connection-pooling).
 
 A Neon connection string includes the role, the compute endpoint hostname, and the database name.
 
 ```text
-postgres://daniel:<password>@ep-restless-rice.us-east-2.aws.neon.tech/neondb
-             ^                          ^                               ^
-             |- <role>                  |- <hostname>                   |- <database>
+postgres://daniel:<password>@ep-mute-rain-952417.us-east-2.aws.neon.tech/neondb
```

Comment:
should we also note the endpoint id in the conn string?

Reply:
I added a note pointing to the definition of the connection string, which describes all the parts.
 GUI applications typically ask for a hostname, so to keep it simple, I don't call out the individual parts of the hostname here.